root/instrument/List.js

```diff
@@ -0,0 +1,47 @@
+const React = require('react');
```

Comment:
You need a copyright header atop this file. Copy it from here, excluding the `@flow` line: https://github.com/metabrainz/musicbrainz-server/blob/master/root/types.js#L2-L6

Replace "MetaBrainz Foundation" with your own real name. Keep one blank line after the header.